packages/wicked/wicked.spec

```diff
@@ -11,7 +11,7 @@
 %bcond_with bootstrap # without
 
 Name: %{_cross_os}wicked
-Version: 0.6.66
+Version: 0.6.68
```

Comment:
Since this release includes sysctl related changes, it'd be good to verify that vmware-k8s-* still works with Cilium for CNI - Cilium has a connectivity health test. (We rely on certain values being set on the network devices that Cilium creates.)

Reply:
I can try testing this

Reply:
Created a 1.21.9 cluster with cluster-api and deployed cilium CNI
Build an OVA off of this branch and launched a few worker nodes with it.
CNI pods are running:
```
ubuntu@ip-172-31-42-218:~/vsphere-testing/conformance-test$ kubectl get pods
NAME                              READY   STATUS    RESTARTS   AGE
....
kube-system   cilium-drtnd                                   1/1     Running             0          41s
kube-system   cilium-ffklj                                   1/1     Running             0          56s
kube-system   cilium-grztq                                   0/1     Running             0          22s
kube-system   cilium-kxwhc                                   1/1     Running             0          17m
kube-system   cilium-operator-588fb5ff4b-g4zt9               1/1     Running             0          17m
....
```

Cilium connectivity tests pass:
```
ubuntu@ip-172-31-42-218:~/vsphere-testing/conformance-test$ ./cilium connectivity test
ℹ️  Monitor aggregation detected, will skip some flow validation steps
⌛ [1-21-9-cluster] Waiting for deployments [client client2 echo-same-node] to become ready...
⌛ [1-21-9-cluster] Waiting for deployments [echo-other-node] to become ready...
⌛ [1-21-9-cluster] Waiting for CiliumEndpoint for pod cilium-test/client-6488dcf5d4-qzwp7 to appear...
⌛ [1-21-9-cluster] Waiting for CiliumEndpoint for pod cilium-test/client2-5998d566b4-k7r2j to appear...
⌛ [1-21-9-cluster] Waiting for CiliumEndpoint for pod cilium-test/echo-other-node-f4d46f75b-29wpd to appear...
⌛ [1-21-9-cluster] Waiting for CiliumEndpoint for pod cilium-test/echo-same-node-745bd5c77-46vmh to appear...
⌛ [1-21-9-cluster] Waiting for Service cilium-test/echo-other-node to become ready...
⌛ [1-21-9-cluster] Waiting for Service cilium-test/echo-same-node to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.32.64:32222 (cilium-test/echo-other-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.32.64:30778 (cilium-test/echo-same-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.129.3:32222 (cilium-test/echo-other-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.129.3:30778 (cilium-test/echo-same-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.16.31:32222 (cilium-test/echo-other-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.16.31:30778 (cilium-test/echo-same-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.64.129:32222 (cilium-test/echo-other-node) to become ready...
⌛ [1-21-9-cluster] Waiting for NodePort 198.19.64.129:30778 (cilium-test/echo-same-node) to become ready...
ℹ️  Skipping IPCache check
⌛ [1-21-9-cluster] Waiting for pod cilium-test/client-6488dcf5d4-qzwp7 to reach default/kubernetes service...
⌛ [1-21-9-cluster] Waiting for pod cilium-test/client2-5998d566b4-k7r2j to reach default/kubernetes service...
🔭 Enabling Hubble telescope...
⚠️  Unable to contact Hubble Relay, disabling Hubble telescope and flow validation: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:4245: connect: connection refused"
ℹ️  Expose Relay locally with:
   cilium hubble enable
   cilium hubble port-forward&
🏃 Running tests...

[=] Test [no-policies]
........................................
[=] Test [allow-all]
....................................
[=] Test [client-ingress]
..
[=] Test [echo-ingress]
....
[=] Test [client-egress]
....
[=] Test [to-entities-world]
......
[=] Test [to-cidr-1111]
....
[=] Test [echo-ingress-l7]
....
[=] Test [client-egress-l7]
..........
[=] Test [dns-only]
..........
[=] Test [to-fqdns]
......
✅ All 11 tests (126 actions) successful, 0 tests skipped, 0 scenarios skipped.

```

Reply:
I was able to build an AMI and successfully run [the connectivity check](https://raw.githubusercontent.com/cilium/cilium/v1.10/examples/kubernetes/connectivity-check/connectivity-check.yaml). My cluster was running Cilium 1.10.3.